config/webpack/webpack.config.js

```diff
@@ -218,15 +221,27 @@ const defaultWebpackConfig = (env = {}, argv = {}) => {
     resolveModules.unshift(path.resolve(rootPath, 'aggregated-translations'));
   }
 
-  const themeOverride = process.env.THEME; // Flexes root theme for theme visual regression testing.
-  const themeFile = ThemeAggregator.aggregate(themeOverride);
+  const defaultTheme = process.env.THEME; // Flexes root theme for theme visual regression testing.
+  const themeConfig = defaultTheme ? { theme: defaultTheme } : null;
+  let themeFile;
+  if (!disableAggregateThemes) {
```

Comment:
This env name makes me think all theme aggregate should be disabled via the aggregator script & the postcss plugin. Should it disable both?

Reply:
The postcss plugin isn't actually doing any aggregating and would eliminate the usefulness of this env. I added it so once we get terra-repos updated we can get the performance boos of removing aggregate-themes without breaking theming tests.